sbt-scalajs-bundler/src/sbt-test/sbt-scalajs-bundler/yarn-interactive/test

```diff
@@ -1 +1 @@
--> fastOptJS::webpack
\ No newline at end of file
+> fastOptJS::webpack
```

Comment:
What makes the test pass now?

Reply:
The version number referred on `build.sbt` was wrong. I just updated it to a valid one